.github/workflows/deploy.yml

```diff
@@ -1,8 +1,6 @@
-name: Deploy to WordPress.org
+name: Deploy to WordPress.org and AWS S3
 on:
-  release:
-    types: [published]
-  workflow_dispatch:
+  push:
```

Comment:
With this change, the action will be executed each time we push to master, this is not the desired behavior, we want to upload to s3 and publish on WordPress when we draft a new release

Reply:
It's change before because of as discuss first we create when migrate with master but make change again and it only work when you release so please check again file change.